lib/transport_web/templates/layout/_footer.html.eex

```diff
@@ -0,0 +1,21 @@
+<footer role="contentinfo">
+  <div class="divider">~</div>
```

Comment:
You might want to use &nbsp;

Reply:
?

Reply:
héhé.
You might want to use `&nbsp;`

Reply:
No, I actually want that symbol, or a text if you have a proposition.

Reply:
Ah, got it. Done.